src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -175,7 +176,7 @@ public static TableModel tableModelFromTable(DatasetTable datasetTable) {
             .datePartitionOptions(dateOptions)
             .intPartitionOptions(intOptions)
             .columns(datasetTable.getColumns().stream()
-                .map(DatasetJsonConversion::columnModelFromDatasetColumn)
+                .map(column -> column.toColumnModel())
```

Comment:
This can also be done using a method reference. You can configure intellij to mark and auto-fix code like this for you.
```suggestion
                .map(Column::toColumnModel)
```